ledger/src/blockstore_processor.rs

```diff
@@ -203,7 +204,11 @@ pub fn execute_batch(
     let first_err = get_first_error(batch, &commit_results);
 
     if let Some(transaction_status_sender) = transaction_status_sender {
-        let transactions = batch.sanitized_transactions().to_vec();
+        let transactions: Vec<SanitizedTransaction> = batch
+            .sanitized_transactions()
+            .iter()
+            .map(|tx| tx.deref().clone())
```

Comment:
Thought we touched on this before, just to be clear, the `deref().clone()` doesn't have meaningful impact on perf, as this function is on hot path.